### PR TITLE
010 gov model

### PR DESCRIPTION
**Pull request by whippit**
initial port with plenty of FIXMEs waiting for other model ports.

### CODE REVIEW

File: synapse/lib/types.py [hunk 1]
```diff
@@ -405,8 +405,13 @@ def _normPyStr(self, valu):
         return self._normPyInt(int(valu, 0))
 
     def _normPyInt(self, valu):
+        if self.opts.get('min') and self.opts.get('min') >= valu:
```
Comment (vertexmc) [inline synapse/lib/types.py]: I would just get the minval once then check if its None, >= valu, return, etc

File: synapse/models/gov/us.py [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-                    ('street', {'ptype': 'str:lwr'}),
-                    ('city', {'ptype': 'str:lwr'}),
-                    ('state', {'ptype': 'str:lwr'}),
-                    ('zip', {'ptype': 'gov:us:zip'}),
-                    ('cc', {'ptype': 'pol:iso2'}),
-                    ('country', {'ptype': 'str:lwr'}),
-                    ('phone0', {'ptype': 'tel:phone'}),
-                    ('phone1', {'ptype': 'tel:phone'}),
+                    # FIXME 010 - need orgs ('name0', ('ou:name', {}), {'doc': 'The name of the organization'}),
+                    ('name1', ('str', {'lower': True}), {'doc': 'Name Part 1'}),
```
Comment (vertexmc) [inline synapse/models/gov/us.py]: I know it was like this before, but do we know what name1 should actually be used for?

File: synapse/tests/test_model_gov_intl.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        ('fakem49', ('gov:intl:un:m49', {}), {'doc': 'A fake int type.'}),
+    ),
+
+    'forms': (
+        ('fakem49', {}, ()),
+    ),
+
+}
+
+class TestModule(s_module.CoreModule):
```
Comment (vertexmc) [inline synapse/tests/test_model_gov_intl.py]: Any reason to have a separate Module here?

File: synapse/models/gov/us.py (hunk 2 above)
Comment (invisig0th) [reply synapse/models/gov/us.py]: The "second part" of the name ( it's in their data structure, which we're basically just mirroring. :( )

File: synapse/tests/test_model_gov_intl.py (hunk 3 above)
Comment (invisig0th) [reply synapse/tests/test_model_gov_intl.py]: Agreed, you could probably just add your fake data model directly to the core.

**Pull request merged**